### ISSUE TITLE
Naming conflict with "sync" module

### ISSUE THREAD

**Issue by Jbalkind** [pulp-platform/common_cells#50]
We're making use of `sync_wedge` and thus `sync` in our design (specifically, it's used in the CLINT from Ariane). The naming used to be nicely prefixed (`pulp_`) to not have any conflicts, but the name `sync` is generic enough that we're getting naming conflicts when working with Amazon's F1 infrastructure.

We tried using `pulp_sync` instead but the interface changed so we had to tinker some and it means we need to keep a fork for little good reason.

Is there any chance you could rename this upstream or adopt a SV package to avoid such conflicts?

**Comment by zarubaf** [comment]
Yeah, that was very unwise naming. We are planning a major bump (which allows us to break API). Is it okay to put this issue down as one of the todos?

**Comment by phsauter** [comment]
Prefixing is planed as part of #287
